plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAMonitor.java

```diff
@@ -32,17 +33,23 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 public class KVMHAMonitor extends KVMHABase implements Runnable {
+
     private static final Logger s_logger = Logger.getLogger(KVMHAMonitor.class);
-    private final Map<String, NfsStoragePool> _storagePool = new ConcurrentHashMap<String, NfsStoragePool>();
+    private final Map<String, NfsStoragePool> _storagePool = new ConcurrentHashMap<>();
+    private final boolean rebootHostAndAlertManagementOnHearbeatTimeout;
 
-    private final String _hostIP; /* private ip address */
+    /* private ip address */
+    private final String _hostIP;
```

Comment:
same here 
```suggestion
    private final String hostIP;
```

Reply:
If the contributor decides to proceed with renaming this variable, I would also suggest removing the commented line `/* private IP address */`, and leave simply `hostIP`, or then something as `hostPrivateIP` or `privateIpAddress`, as this is expected to be the `HostVO.privateIpAddress`.